### ISSUE TITLE
tmt prepare doesn't seem to work with dnf5

### ISSUE THREAD

**Issue by mrc0mmand** [teemtee/tmt#2163]
Hey!

One of our CI jobs that runs on latest Rawhide started failing once dnf5 got into Rawhide. It can be easily reproduced with:

```
# mkdir test
# cd test
# tmt init
Tree '/root/test' initialized.
To populate it with example content, use --template with mini, base or full.
# tmt tests create -t beakerlib foo
Directory '/root/test/foo' created.
Test metadata '/root/test/foo/main.fmf' created.
Test script '/root/test/foo/test.sh' created.
```

```
# tmt run discover --how fmf provision --how local prepare 
/var/tmp/tmt/run-013

/plans/default
    discover
        how: fmf
        directory: /root/test
        summary: 1 test selected
    provision
        how: local
        multihost name: default-0
        arch: x86_64
        distro: Fedora Linux 39 (Rawhide Prerelease)
        summary: 1 guest provisioned
    prepare
        queued push task #1: push on default-0
        
        push task #1: push on default-0
    
        queued prepare task #1: requires on default-0
        
        prepare task #1: requires on default-0
        how: install
        summary: Install required packages
        name: requires
        fail: Package manager "None" is not supported.

prepare step failed

The exception was caused by 1 earlier exceptions

Cause number 1:

    Package manager "None" is not supported.
```

```
# rpm -q tmt dnf5
tmt-1.24.1-1.fc39.noarch
dnf5-5.0.14-1.fc39.x86_64
```

With dnf4 this seems to work fine as expected.

**Comment by mrc0mmand** [comment]
And it looks like this is being resolved in https://github.com/teemtee/tmt/pull/2162.

**Comment by lukaszachy** [comment]
Fixed tmt will be available from copr psss/tmt once #2162 is merged.

If one cannot / doesn't want to use 1.25.dev yet, there is already 1.24.1-3 built with cherrypicked fixes in the same copr repo (https://copr.fedorainfracloud.org/coprs/psss/tmt/build/6104020/)

However one has to specify the version explicitely, otherwise newer NVR wins.
```
sudo dnf copr enable -y psss/tmt
sudo dnf update 'tmt-1.24.1-3*'
```
